paasta_tools/autoscaling/autoscaling_service_lib.py

```diff
@@ -268,17 +268,16 @@ def deserialize_historical_load(historical_load_bytes):
     return historical_load
 
 
-async def get_json_body_from_service(host, port, endpoint, timeout=2):
-    async with aiohttp.ClientSession(
-        conn_timeout=timeout, read_timeout=timeout
-    ) as session:
-        async with session.get(
-            f"http://{host}:{port}/{endpoint}", headers={"User-Agent": get_user_agent()}
-        ) as response:
-            return await response.json()
+async def get_json_body_from_service(host, port, endpoint, session):
+    async with session.get(
+        f"http://{host}:{port}/{endpoint}", headers={"User-Agent": get_user_agent()}
+    ) as response:
+        return await response.json()
 
 
-async def get_http_utilization_for_a_task(task, service, endpoint, json_mapper):
+async def get_http_utilization_for_a_task(
+    task, service, endpoint, json_mapper, session
```

Comment:
I will give you 5 cents if you add instance here and in the log at line 303

Reply:
and line 308

Reply:
added